conf.d/docker.yaml.example

```diff
@@ -1,5 +1,5 @@
 # Warning
-# The user running the Datadog Agent (usually "dd-agent") must be part of the "docker" group
+# The user running the StackState Agent (usually "dd-agent") must be part of the "docker" group
```

Comment:
Good one, there were still a lot of these - added them to `./sts-check-naming.sh` and fixed them